cg/meta/demultiplex/delete_demultiplex_api.py

```diff
@@ -260,7 +274,9 @@ def delete_flow_cell(
             )
             self.delete_flow_cell_cgstats()
             self.delete_flow_cell_housekeeper()
+            self.delete_flow_cell_sample_lane_sequencing_metrics()
```

Comment:
```suggestion
```
is not needed here since deleted a flow cell in statusDB cascades to sample lane sequencing metrics